<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>401</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboboxDevices"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineeditTitle"/>
     </item>
     <item>
      <widget class="QTabWidget" name="tabwidgetFunctions">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tabLinkPush">
        <property name="text" stdset="0">
         <string/>
        </property>
        <attribute name="title">
         <string>Link</string>
        </attribute>
        <widget class="QPlainTextEdit" name="plaintexteditLink">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>401</width>
           <height>291</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabNotePush">
        <attribute name="title">
         <string>Note</string>
        </attribute>
        <widget class="QPlainTextEdit" name="plaintexteditNote">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>401</width>
           <height>291</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>140</y>
           <width>160</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout"/>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSubmit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>380</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
   </widget>
   <addaction name="menuOption"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
